### ISSUE TITLE
Incorrect comment in [intro.execution] p. 12

### ISSUE THREAD

**Issue by RazvanAM** [cplusplus/draft#1706]
In [this paragraph](http://eel.is/c++draft/intro.execution#12) there is the following code sample:

`S s1(1);                   // full-expression is call of S​::​S(int)`

Why is the full-expression the call to the constructor and not the entire construct (init-declarator) itself? (In the same paragraph (p 12.3) it says that an init-declarator is a full-expression.)

**Comment by jensmaurer** [comment]
Where's the difference in this case?

**Comment by RazvanAM** [comment]
There is no difference in behaviour, I think it is just a minor inconsistency in wording. 
I tracked the evolution of the paragraph describing full-expressions and I discovered that the code sample that I cited was written even before the passage: "If a language construct is defined to produce an implicit call of a function, a use of the language construct is considered to be an expression for the purposes of this definition" was added. And those samples remained the same, even after other changes were made to the paragraph. I actually sent you an e-mail regarding this (at Jens.Maurer@gmx.net).
